lib/util/trigger.php

```diff
@@ -9,39 +9,129 @@ function parseOperand($mem)
 
     $type = '';
     switch ($mem[0]) {
-        case 'd': case 'D': $type = 'Delta'; $mem = substr($mem, 1); $max--; break;
-        case 'p': case 'P': $type = 'Prior'; $mem = substr($mem, 1); $max--; break;
-        case 'b': case 'B': $type = 'BCD'; $mem = substr($mem, 1); $max--; break;
-        case '~':           $type = 'Inverted'; $mem = substr($mem, 1); $max--; break;
+        case 'd':
+        case 'D':
+            $type = 'Delta';
+            $mem = substr($mem, 1);
+            $max--;
+            break;
+        case 'p':
+        case 'P':
+            $type = 'Prior';
+            $mem = substr($mem, 1);
+            $max--;
+            break;
+        case 'b':
+        case 'B':
+            $type = 'BCD';
+            $mem = substr($mem, 1);
+            $max--;
+            break;
+        case '~':
+            $type = 'Inverted';
+            $mem = substr($mem, 1);
+            $max--;
+            break;
     }
 
     $size = '';
     if ($max > 3 && $mem[0] == '0' && $mem[1] == 'x') {
         switch ($mem[2]) {
-            case 'h': case 'H': $size = '8-bit'; break;
-            case ' ':           $size = '16-bit'; break;
-            case 'x': case 'X': $size = '32-bit'; break;
-
-            case 'm': case 'M': $size = 'Bit0'; break;
-            case 'n': case 'N': $size = 'Bit1'; break;
-            case 'o': case 'O': $size = 'Bit2'; break;
-            case 'p': case 'P': $size = 'Bit3'; break;
-            case 'q': case 'Q': $size = 'Bit4'; break;
-            case 'r': case 'R': $size = 'Bit5'; break;
-            case 's': case 'S': $size = 'Bit6'; break;
-            case 't': case 'T': $size = 'Bit7'; break;
-            case 'l': case 'L': $size = 'Lower4'; break;
-            case 'u': case 'U': $size = 'Upper4'; break;
-            case 'k': case 'K': $size = 'BitCount'; break;
-            case 'w': case 'W': $size = '24-bit'; break;
-            case 'g': case 'G': $size = '32-bit BE'; break;
-            case 'i': case 'I': $size = '16-bit BE'; break;
-            case 'j': case 'J': $size = '24-bit BE'; break;
```

Comment:
I prefer the grid look of the before (lowercase in first column, uppercase in second, result in third), but I understand the desire for consistency.

Reply:
Same, I think keeping the column formatting in this particular case would be preferable. I'll check if I can exclude this block from the rules.

Reply:
Restored formatting 👍